docs/content/docs/dev/datastream/sinks.md

```diff
@@ -93,36 +93,36 @@ The `Sink` needs to be serializable. All configuration should be validated eager
 
 ### Custom sink topology
 
-For advanced developers, they may want to specify their own sink operator topology(A structure composed of a series of operators), such as collecting `committables` to one subtask and processing them together, or performing operations such as merging small files after `Committer`. Flink provides the following interfaces to allow expert users to customize the sink operator topology.
+For advanced developers and expert users, they may want to specify their own sink operator topology (A structure composed of a series of operators), such as collecting `committables` to one subtask and processing them together, or performing operations such as merging small files after `Committer`. Flink provides the following interfaces to allow expert users to customize the sink operator topology.
 
 #### SupportsPreWriteTopology
 
-`SupportsPreWriteTopology` interface Allows expert users to implement a custom operator topology before `SinkWriter`, which can be used to process or redistribute the input data. For example, sending data of the same partition to the same SinkWriter of Kafka or Iceberg.
+`SupportsPreWriteTopology` interface allows expert users to implement a custom operator topology before `SinkWriter`, which can be used to process or redistribute the input data. For example, sending data of the same partition to the same Kafka or Iceberg SinkWriter.
 
-The following figure shows the operator topology of using {{< gh_link file="flink-runtime/src/main/java/org/apache/flink/streaming/api/connector/sink2/SupportsPreWriteTopology.java" name="SupportsPreWriteTopology" >}}:
+The following figure shows the operator topology using {{< gh_link file="flink-runtime/src/main/java/org/apache/flink/streaming/api/connector/sink2/SupportsPreWriteTopology.java" name="SupportsPreWriteTopology" >}}:
 
 {{< img src="/fig/dev/datastream/SupportsPreWriteTopology.png" class="center" >}}
 
-In the figure above, user add a `PrePartition` and `PostPartition` operator in the `SupportsPreWriteTopology` topology, and redistribute the input data to the `SinkWriter`.
+In the figure above, users add a `PrePartition` and `PostPartition` operator in the `SupportsPreWriteTopology` topology, and redistribute the input data to the `SinkWriter`.
```

Comment:
nit: a `PrePartition` and `PostPartition` operator in -> `PrePartition` and `PostPartition` operators to